PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1811,6 +1833,37 @@ bool Notepad_plus::findInCurrentFile()
 	return true;
 }
 
+void Notepad_plus::showIgnoredFilesMessage(ProcessOperation op, const std::vector<generic_string>& ignoredFiles)
+{
+	generic_string message;
+	switch (op)
+	{
+	case ProcessFindAll:
+		message += ignoredFiles.size() > 1 ?
+			_T("Below files were ignored while finding in folder.\r\n\r\n") :
+			_T("Below file was ignored while finding in folder.\r\n\r\n");
+		break;
+	case ProcessReplaceAll:
+		message += ignoredFiles.size() > 1 ?
+			_T("Below files were ignored while replacing in folder.\r\n\r\n") :
+			_T("Below file was ignored while replacing in folder.\r\n\r\n");
+		break;
+	default:
+		message += ignoredFiles.size() > 1 ?
+			_T("Below files were ignored.\r\n\r\n") :
+			_T("Below file was ignored.\r\n\r\n");
+		break;
+	}
+
+	for (auto file : ignoredFiles)

```

Comment:
either `auto&` or `message += std::move(file)` - preferably the former. Props for going with a foreach though, they are much underused in the codebase (iterating through `std::vector` with an index and comparing it against `size()` makes me shiver a bit).

Reply:
Will update as ```const auto&```.